certbot/compat/misc.py

```diff
@@ -17,6 +17,19 @@
 from certbot import errors
 from certbot.compat import os
 
+
+# UMASK_FOR_PRIVATE_KEY_PERMISSIONS defines what are the permissions flags to keep
+# when transferring the permissions from an old private key to a new one.
+if POSIX_MODE:
+    # On Linux, we keep read/write/execute permissions
+    # for group and read permissions for everybody.
+    UMASK_FOR_PRIVATE_KEY_PERMISSIONS = stat.S_IRGRP | stat.S_IWGRP | stat.S_IXGRP | stat.S_IROTH
+else:
+    # On Windows, the mode returned by os.stat is not reliable (all bits are always set.
```

Comment:
nit: It's not exactly true that all bits are always set. I'd remove the parenthetical here and just say:

> ...is not reliable so we do not...

Reply:
Done.